src/main/resources/assets/fuelinfo/lang/ru_ru.json

```diff
@@ -0,0 +1,7 @@
+{
+  "message.fuelinfo.furnace.items": "§7Эта печь может переплавить §e%d §7предмет(а/ов)",
+  "message.fuelinfo.furnace.stacks": "§7Эта печь может переплавить §e%d §7стак(а/ов)",
+  "message.fuelinfo.furnace.both": "§7Эта печь может переплавить §e%d §7стак(а/ов) и §e%d §7предмет(а/ов)",
+
+  "message.fuelinfo.brewing_stand": "§7This Brewing Stand can produce §e%d §7potions"
```

Comment:
Can you translate this line as well? 🙂

Reply:
I'm at work right now, will do it later. Don't know how I missed this line 😄

Reply:
Fixed it, sorry!